pkg/server/server.go

```diff
@@ -61,7 +61,7 @@ func New(cfg Config) (*server.Server, error) {
 		Version:               cfg.Version,
 		HTTPSListenPort:       7443,
 		LongRunningVerbs:      []string{"watch", "proxy"},
-		LongRunningResources:  []string{"exec", "proxy", "log", "registryport", "port", "push", "pull", "portforward"},
+		LongRunningResources:  []string{"exec", "proxy", "log", "registryport", "port", "push", "pull", "portforward", "copy"},
```

Comment:
This was needed in order to avoid hitting context deadlines.

Reply:
Great job figuring this out!